packages/target-electron/src/index.ts

```diff
@@ -149,9 +149,10 @@ Promise.all([
     log.critical('Fatal Error during init', error)
     dialog.showErrorBox(
       'Fatal Error during init',
-      '' +
-        error +
-        '\n\nAlso make sure you are not trying to run multiple instances of deltachat.'
+      `[DC Version: ${BuildInfo.VERSION}]
```

Comment:
While we're at it, why not add the core version here as well?

Reply:
I said "not so easy", not impossible. sure you could run the rpc server with `--version`, but it's again code that could fail. especially since most errors in this dialog were "could not find or start core/rpc-server" in the past (catching this scenario is the main purpose of these dialogs, core crashes are very rare these days).